var/spack/repos/builtin/packages/fairlogger/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+#   Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright 2020 GSI Helmholtz Centre for Heavy Ion Research GmbH,
+#   Darmstadt, Germany
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fairlogger(CMakePackage):
+    """Lightweight and fast C++ Logging Library"""
+
+    homepage = 'https://github.com/FairRootGroup/FairLogger'
+    url = "https://github.com/FairRootGroup/FairLogger/archive/v1.2.0.tar.gz"
+    git = 'https://github.com/FairRootGroup/FairLogger.git'
+    maintainers = ['dennisklein', 'ChristianTackeGSI']
+    # generator = 'Ninja'
+
+    version('dev', branch='dev')
```

Comment:
```suggestion
    version('develop', branch='dev')
```
Spack has special handling for `master` and `develop` versions so that it knows they are newer than all of the numerical releases.